### PR TITLE
update priority

### PR DESCRIPTION
**Pull request by kylin321370**

### CODE REVIEW

File: plugins/huaweicloud/src/main/java/org/eclipse/osc/orchestrator/plugin/huaweicloud/AtomBuilder.java [hunk 1]
```diff
@@ -88,6 +91,18 @@ private boolean waitSubBuilders() {
...
+        List<AtomBuilder> subBuilderList = new ArrayList<>();
+        for (AtomBuilder subBuilder : subBuilders) {
+            for (Integer priority = 1; priority <= PRIORITY_LEVEL; priority++) {
+                if (subBuilder.getPriority()==priority){
+                    subBuilderList.add(subBuilder);
+                }
+            }
+        }
+        return subBuilderList;
+    }
```
Comment (baixinsui) [inline plugins/huaweicloud/src/main/java/org/eclipse/osc/orchestrator/plugin/huaweicloud/AtomBuilder.java:104]: This sort traversal time complexity is too high. The following method is recommended：list.stream().sorted(Comparator.comparing(AtomBuilder::getPriority)).collect(Collectors.toList());

Comment (baixinsui) [inline plugins/huaweicloud/src/main/java/org/eclipse/osc/orchestrator/plugin/huaweicloud/AtomBuilder.java:104]: This sort traversal time complexity is too high. The following method is recommended：list.stream().sorted(Comparator.comparing(AtomBuilder::getPriority)).collect(Collectors.toList());

**Comment by baixinsui** [comment]
This sort traversal time complexity is too high. The following method is recommended：list.stream().sorted(Comparator.comparing(AtomBuilder::getPriority)).collect(Collectors.toList());

**Comment by iskey** [comment]
We should use two different builders to make it simple. For now, the priority resolution is dropped.

**Pull request closed without merging**